src/services/WithdrawalService.ts

```diff
@@ -96,39 +103,43 @@ export default class WithdrawalService {
     }
 
     static async withdraw(assetPool: AssetPoolType, withdrawal: WithdrawalDocument) {
-        // if (NETWORK_ENVIRONMENT === 'dev' || NETWORK_ENVIRONMENT === 'prod') {
-        //     const tx = await InfuraService.send(
-        //         assetPool.address,
-        //         'withdrawPollFinalize',
-        //         [withdrawal.withdrawalId],
-        //         assetPool.network,
-        //     );
-
-        //     withdrawal.transactions.push(String(tx._id));
-
-        //     return await withdrawal.save();
-        // } else {
-        try {
-            const { tx, receipt } = await TransactionService.send(
+        if (NETWORK_ENVIRONMENT === 'dev' || NETWORK_ENVIRONMENT === 'prod') {
```

Comment:
Better to make this independent of the `NETWORK_ENVIRONMENT` env var.. and set it in the network configuration or as a separate env var..